CefSharp.WinForms.Example/BrowserForm.cs

```diff
@@ -230,5 +233,36 @@ private void CloseDevToolsMenuItemClick(object sender, EventArgs e)
                 control.Browser.CloseDevTools();
             }
         }
+
+        private void zoomInToolStripMenuItem_Click(object sender, EventArgs e)
```

Comment:
Rename to `ZoomInToolStripMenuItemClick` for consistency, other two click handlers also.